pkg/sql/copy/copy_test: TestLargeCopy failed

pkg/sql/copy/copy_test.TestLargeCopy [failed](https://mesolite.cluster.engflow.com/invocations/default/5a077750-a4a7-4b94-bb93-2872d66df9ce?testReportRun=8&testReportShard=3&testReportAttempt=1#targets-Ly9wa2cvc3FsL2NvcHk6Y29weV90ZXN0) on release-26.1.1-rc @ [5abbbd24fcc2c7873fb0f985ccea68b7abaa6afe](https://github.com/cockroachdb/cockroach/commits/5abbbd24fcc2c7873fb0f985ccea68b7abaa6afe):


```
=== RUN   TestLargeCopy
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestLargeCopy3293513759
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:182: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    copy_test.go:782: dial tcp 127.0.0.1:43849: connect: connection refused
    panic.go:615: -- test log scope end --
test logs left over in: outputs.zip/logTestLargeCopy3293513759
--- FAIL: TestLargeCopy (95.60s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=8</code>
 - <code>shard=3</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #164723 pkg/sql/copy/copy_test: TestLargeCopy failed [C-test-failure O-robot T-sql-queries branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLargeCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-61238

--- a/pkg/sql/copy/BUILD.bazel
+++ b/pkg/sql/copy/BUILD.bazel
@@ -10,6 +10,9 @@ go_test(
         "main_test.go",
     ],
     data = glob(["testdata/**"]),
+    exec_properties = {
+        "test.Pool": "large",
+    },
     shard_count = 4,
     deps = [
         "//pkg/base",

